### PR TITLE
[DO NOT MERGE] Release 1.0.0

### PR DESCRIPTION
**Pull request by style95**
This is to release OpenWhisk core 1.0.0 and fix tags from the `nightly` to a fixed tag.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [ ] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [ ] I reviewed the [style guides](https://github.com/apache/openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by style95** [comment]
This PR would not be merged.
This is a branch in the Apache repo and would preserve a snapshot for the 1.0.0 version.
image

Reference: https://github.com/apache/openwhisk-release/blob/master/docs/release_instructions.md#preparing-to-make-a-release

File: ansible/files/runtimes.json [hunk 1]
```diff
@@ -262,7 +248,7 @@
                 "image": {
                     "prefix": "openwhisk",
                     "name": "action-ballerina-v0.990.2",
-                    "tag": "nightly"
+                    "tag": "nightly" // Todo: Need to use a fixed tag after releasing the runtime
```
Comment (style95) [inline ansible/files/runtimes.json:251]: I noticed there is no released version of this runtime and it's even not listed in the website.
Should we release this runtime first?

**Review by style95 — COMMENTED** [review]
@dgrove-oss 
So should we build this branch and push the images for controllers, invokers, standalone, etc and update this branch again to apply the fixed tags for them?

File: ansible/files/runtimes.json [hunk 2]
```diff
@@ -71,27 +71,13 @@
             }
         ],
         "python": [
-            {
```
Comment (style95) [inline ansible/files/runtimes.json:74]: Python2 would be officially gone from the 1.0.0.

File: ansible/files/runtimes.json (hunk 1 above)
Comment (dgrove-oss) [reply ansible/files/runtimes.json:251]: I think we should probably drop ballerina from 1.0.0 branch, but I will defer to @rabbah.  I think ballerina 1.2.x is out now, so if we did want to support it, the "right" thing to do is update the runtime to a newer version of ballerina, then release it, then use that tag.   I don't think we should hold 1.0.0 of OpenWhisk to do that.

**Comment by dgrove-oss** [comment]
I will see if I can manage to convince the Jenkins job to build the right core Docker images from the 1.0.0 branch.  Might not have time until later tonight.

File: ansible/files/runtimes.json [hunk 3]
```diff
@@ -205,7 +191,7 @@
                 "image": {
                     "prefix": "openwhisk",
                     "name": "action-golang-v1.11",
```
Comment (style95) [inline ansible/files/runtimes.json:193]: Should we keep go-v1.11 along with go-v1.15?
Reference: [GO-EOL](https://endoflife.date/go)

Comment (dgrove-oss) [reply ansible/files/runtimes.json:193]: Good point.  We should drop go-v1.11 imo.

**Comment by style95** [comment]
Now, this branch includes these two new commits.

image

And I think it is finally ready to be released.

**Review by rabbah — APPROVED** [review]

**Comment by style95** [comment]
closing as the 1.0.0 version is released.

**Pull request closed without merging**